apollo.sh

```diff
@@ -221,6 +222,9 @@ function main() {
         clean)
             env ${APOLLO_ENV} bash "${APOLLO_ROOT_DIR}/scripts/apollo_clean.sh" "$@"
             ;;
+        release)
+            env ${APOLLO_ENV} bash "${APOLLO_ROOT_DIR}/scripts/apollo_release.sh" "$@"
+            ;;
```

Comment:
Implement `./apollo.sh release` subcommand. Command completion should be provided in later PRs. Maybe @changsh726 can help.